pyspark/Dockerfile

```diff
@@ -15,13 +11,29 @@ LABEL io.k8s.description="Platform for building a radanalytics pyspark app" \
       io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
       io.openshift.tags="builder,radanalytics,pyspark"
 
-ENV APP_ROOT /opt/app-root
+USER root
+RUN yum install -y epel-release tar java && \
+    yum clean all
+
+RUN cd /opt && \
+    curl https://dist.apache.org/repos/dist/release/spark/spark-2.0.0/spark-2.0.0-bin-hadoop2.7.tgz | \
+        tar -zx && \
+    ln -s spark-2.0.0-bin-hadoop2.7 spark
+
+ENV PATH=$PATH:/opt/spark/bin
+ENV SPARK_HOME=/opt/spark
+
+# Add scripts used to configure the image
+COPY scripts /tmp/scripts
+
+# Custom scripts
+RUN bash -x /tmp/scripts/spark/install && rm -rf /tmp/scripts
```

Comment:
the install script does things needed for a spark master/worker, i don't think it is needed here